src/page/VariantStore.ts

```diff
@@ -65,16 +66,18 @@ export class VariantStore {
     @observable public variant: string = '';
     @observable public selectedTranscript: string = '';
 
+    public annotationQueryFields() {
+        const fields = ['annotation_summary', 'my_variant_info'];
+        // add more fields based on config
+        SHOW_MUTATION_ASSESSOR && fields.push('mutation_assessor');
+        return fields;
```

Comment:
maybe add all the fields to the configDefaults? Then you don't have to do two checks for `SHOW_MUTATION_ASSESSOR` (i.e. in `annotationQueryFields` and in `jsonButton`). Rather you can use the fields config variable

Reply:
@inodb not sure I understand correctly, I add these to `configDefaults`
```
export const ANNOTATION_QUERY_FIELDS = SHOW_MUTATION_ASSESSOR
    ? ['annotation_summary', 'my_variant_info', 'mutation_assessor']
    : ['annotation_summary', 'my_variant_info'];
export const JSON_RESPONSE_QUERY_FIELDS = SHOW_MUTATION_ASSESSOR
    ? '?fields=hotspots%2Cmy_variant_info%2Cptms%2Cannotation_summary%2Cmutation_assessor'
    : '?fields=hotspots%2Cmy_variant_info%2Cptms%2Cannotation_summary';
```
 Is this what you were suggesting?